main.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/go-redis/redis/v8"
 	"github.com/urfave/cli/v2"
+	"github.com/spf13/viper"
```

Comment:
Don't we need to get this package and update go.mod and go.sum files accordingly?